packages/react-magma-docs/src/pages/design/selection-controls.mdx

```diff
@@ -26,7 +27,7 @@ Selection controls allow users to complete tasks that involve making choices suc
     <p>Selection controls</p>
   </figcaption>
 </figure>
-<hr class="article-divider" />
+<Divider />
```

Comment:
oh man! a divider without the `span.horizontal-spacer` 😂

Reply:
Womp womp.  It _should_ probably be consistent though -- not sure if that was intentional.